### ISSUE TITLE
[maven] The maven-plugin is not marked as ThreadSafe

### ISSUE THREAD

**Issue by bmarwell** [jreleaser/jreleaser#1591]
### Task List

- [X] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Use JReleaser-Maven-Plugin in a multi-module-build to create a java archive (or execute any other task)
2. start the build using either `mvn -T1C` or `mvnd`
3. See the output

### Expected Behaviour

Nor warning issued by Maven

### Actual Behaviour

```
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but this         *
[WARNING] * project contains the following plugin(s) that have goals not  *
[WARNING] * marked as thread-safe to support parallel execution.          *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against Apache Maven.                           *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked as thread-safe in myproject :: main app (main):
[WARNING]   org.jreleaser:jreleaser-maven-plugin:1.8.0
[WARNING]   org.moditect:moditect-maven-plugin:1.0.0.Final
[WARNING] 
[WARNING] Enable debug to see precisely which goals are not marked as thread-safe.
[WARNING] *****************************************************************
```

### Environment Information

- **Operating System**: Mac, Linux
- **JReleaser Version:** 1.8.0
- **JDK Version:** 21

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

not relevant

**Comment by aalmiray** [comment]
What does it mean for a plugin to be marked as threadsafe?

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.11.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.11.0))
